vg/vgimg/vgimg_test.go

```diff
@@ -0,0 +1,43 @@
+package vgimg_test
+
+import (
+	"bytes"
+	"log"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/plotter"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+	"github.com/gonum/plot/vg/vgimg"
+)
+
+func TestIssue179(t *testing.T) {
+	scatter, err := plotter.NewScatter(plotter.XYs{{1, 1}, {0, 1}, {0, 0}})
+	if err != nil {
+		log.Fatal(err)
+	}
+	p, err := plot.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	p.Add(scatter)
+	p.HideAxes()
+
+	c := vgimg.JpegCanvas{Canvas: vgimg.New(vg.Inch, vg.Inch)}
+	p.Draw(draw.New(c))
+	b := bytes.NewBuffer([]byte{})
+	n, err := c.WriteTo(b)
+	if err != nil {
+		t.Error(err)
+	}
+	if n != int64(len(issue179Answer)) {
```

Comment:
This is redundant. bytes.Equal does this already and I'm not sure how valuable the differentiation is here.